src/crafting.h

```diff
@@ -69,6 +69,8 @@ struct recipe {
         bool hidden;
     };
     std::vector<bookdata_t> booksets;
+    std::set<std::string> flags;
```

Comment:
Why not add two proper bools to the class? That would not only be faster, but also safer regarding spelling mistakes (one might accidentally check for a "BLINDEASY" flag).